nephio/optional/nephio-stock-repos/repo-free5gc-packages.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: config.porch.kpt.dev/v1alpha1
+kind: Repository
+metadata:
+  name: free5gc-packages
```

Comment:
Should we update the names of these and all the related references to them? Maybe all the Porch Repository objects that point to the nephio-project/catalog directory should start with "catalog" or even "nephio-catalog":

- `nephio-catalog-workloads-free5gc`
- `nephio-catalog-workloads-oai`
- `nephio-catalog-nephio-core`
- `nephio-catalog-nephio-optional`
- `nephio-catalog-infra-capi`

etc.? Or just `catalog-workloads-free5gc`, etc. if you think it's too verbose/redundant.

Reply:
Okay we can change the naming, I thought we will register each sub directory inside the `nephio-core` also. So you just want to register `nephio-core` and `nephio-optional`. I think that is even better else there will be many definitions. Then in that case for workload we can have just one rather than having `oai` and `free5gc`.

Reply:
Btw this way the repo name would be `catalog-nephio-core` rather than `nephio-core` (it will map to `nephio/core` directory) I suppose.

Reply:
Yeah. We should not register *package* directories as repositories, just their parent directory.